docs/src/main/paradox/deploy.md

```diff
@@ -4,6 +4,12 @@ You can deploy an Akka gRPC application just like you would any other JVM-based
 
 Remember that the cleartext HTTP/2 "h2c with prior knowledge" protocol is not compatible with HTTP/1.1, so if your infrastructure uses any proxies they must either understand this protocol or support generic TCP connections.
 
+## Service gRPC over TLS
+
+To deploy your gRPC service over a TLS connection you will have to use an @apidoc[akka.http.(javadsl|scaladsl).HttpsConnectionContext] as described in the [Akka-HTTP documentation](https://doc.akka.io/docs/akka-http/10.1/server-side/server-https-support.html).
+
+When using TLS and running your application on a JDK under version 9 you also need to provide an external ALPN implementation. Follow the [section on ALPN](https://doc.akka.io/docs/akka-http/10.1/server-side/http2.html#application-layer-protocol-negotiation-alpn-) in the Akka-HTTP docs for more details.
+
```

Comment:
I'm linking to the `10.1` version on purpose, when we upgrade the dependency we'll have to update this too. 
The reason is the `HttpsConnectionContext.https` overloads used to enable HTTP/2 are deprecated.